src/i18n/messages-fr.po

```diff
@@ -0,0 +1,875 @@
+
+msgid "Could not load PDF: Missing PDF file."
+msgstr "Impossible de charger le PDF : le fichier PDF est manquant"
+
+#, python-format
+msgid "An error occurred while loading the PDF (%s)."
+msgstr "Une erreur est survenue lors du chargement du PDF (%s)."
+
+msgid "An unknown error occurred while loading the PDF."
+msgstr "Une erreur inconnue est survenue lors du chargement du PDF"
+
+#, python-format
+msgid "An error occurred while loading the PDF page (%s)."
+msgstr "Une erreur est survenue lors du chargement du PDF (%s)."
+
+msgid "An unknown error occurred while loading the PDF page."
+msgstr "Une erreur inconnue est survenue lors du chargement du PDF (%s)."
+
+#, python-format
+msgid "An error occurred while rendering the PDF page (%s)."
+msgstr "Une erreur est survenue lors du rendu du PDF (%s)."
+
+msgid "An unknown error occurred while rendering the PDF page."
+msgstr "Une erreur inconnue est survenue lors du rendu du PDF"
+
+msgid "Only PDF documents and OpenDocument files can be shared at this time."
+msgstr ""
+"Pour l'instant, seuls les documents au format pdf et open documents peuvent "
+" être partagés"
+
+#, python-format
+msgid "Failed to start screen sharing (%s)."
+msgstr "Impossible de démarrer le partage d'écran (%s)."
+
+msgid ""
+"Permission to start screen sharing was denied. Make sure to have enabled "
+"screen sharing access for your browser. Copy chrome://flags/#enable-"
+"usermedia-screen-capture and open it with your browser and enable the "
+"flag on top. Then restart the browser and you are ready to go."
+msgstr ""
+"La permission de démarrer le partage d'écran a été refusée. Soyez sûr "
+" que vous avez activez le partage pour votre navigateur. Copiez chrome://flags/#enable-"
+"usermedia-screen-capture dans votre barre d'adresse et activez le, puis "
+"redémarrez votre navigateur"
+
+msgid "Permission to start screen sharing was denied."
+msgstr "La permission de partager l'écran a été refusé."
+
+msgid "Use browser language"
+msgstr "Utiliser la langue du navigateur"
+
+msgid "Meet with me here:"
+msgstr "Rejoins moi là:"
+
+msgid "Room name"
+msgstr "Nom de la salle"
+
+msgid ""
+"The request contains an invalid parameter value. Please check the URL of "
+"the video you want to share and try again."
+msgstr ""
+"La requête contient des paramètres invalides. Merci de vérifier l'URL "
+" de la vidéo que vous voulez partager"
+
+msgid ""
+"The requested content cannot be played in an HTML5 player or another "
+"error related to the HTML5 player has occurred. Please try again later."
+msgstr ""
+"Le contenu requis ne peux pas être pas lu par le lecteur HTML5, ou une autre"
+"erreur en relation avec le lecteur HTML5 est survenu. Merci de réessayer plus tard"
+
+msgid ""
+"The video requested was not found. Please check the URL of the video you "
+"want to share and try again."
+msgstr "Cette vidéo n'a pas été trouvée. Merci de vérifier l'URL de la vidéo."
+" que vous souhaitez partager en réessayez"
```

Comment:
Either start the new sentence with an uppercase character, or remove the previous dot if it's the same sentence. Also missing trailing dot.